station-api-base/src/main/java/net/modificationstation/stationapi/api/util/Namespace.java

```diff
@@ -144,6 +150,22 @@ public final class Namespace implements Comparable<@NotNull Namespace> {
         });
     }
 
+    private static boolean canonicalURIEquals(URI a, URI b) {
+        if (a.equals(b))
+            return true;
+        return canonicalPathEquals(Paths.get(a), Paths.get(b));
```

Comment:
Okay, this does not work completely yet, apparently. In my tests yesterday, everything worked fine, but now this is spitting errors sometimes...
It's a `FileSystemNotFoundException`, but I cannot explain why - since the file obviously exists (checked manually while debugging).
I will leave this PR open as I may or may not have time to work on this again... If someone wants to fix this in the mean time, go ahead!